Makefile

```diff
@@ -1,16 +1,21 @@
-build: build-cedar-14 build-cedar-10
+build: build-heroku-16 build-cedar-14 build-cedar-10
+
+build-heroku-16:
+	@echo "Building Nginx in docker for heroku-16..."
+	@docker run -v $(shell pwd):/buildpack --rm -it -e "STACK=heroku-16" -w /buildpack heroku/heroku:16 bash -c 'apt-get update && apt-get install -y build-essential zlib1g-dev && scripts/build_nginx /buildpack/bin/nginx-heroku-16'
```

Comment:
Instead of trying to install build dependencies on base runtime image you can use `-build` version of this instead: [`heroku/heroku:16-build`][1].

[1]: https://devcenter.heroku.com/articles/heroku-16-stack#heroku-16-docker-image

Reply:
Thank you for the information!
It is nice to know.